### PR TITLE
Use information cover cache

### PR DESCRIPTION
**Pull request by TotalCaesar659**
Partly fixes https://github.com/sharkwouter/minigalaxy/issues/367.

### CODE REVIEW

**Comment by sharkwouter** [comment]
Thanks for this PR, this is a great feature to have. I did find an issue with it, though, even though it does seem to mostly work.

When opening information for a game which does not have the cover stored yet, I see the following exception pop up in the terminal:

```
Exception in thread Thread-379:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/wouter/Personal/minigalaxy/minigalaxy/download_manager.py", line 86, in __download_file
    result = self.download_operation(download, start_point, download_mode)
  File "/home/wouter/Personal/minigalaxy/minigalaxy/download_manager.py", line 134, in download_operation
    file_size = int(download_request.headers.get('content-length'))
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

File: minigalaxy/ui/window.py [hunk 1]
```diff
@@ -66,6 +66,10 @@ def __init__(self, name="Minigalaxy"):
         if not os.path.exists(THUMBNAIL_DIR):
             os.makedirs(THUMBNAIL_DIR, mode=0o755)
 
+        # Create the covers directory
+        if not os.path.exists(COVER_DIR):
```
Comment (sharkwouter) [inline minigalaxy/ui/window.py:70]: Could you move the creation of empty directories for thumbnails and covers to a ``make_directories`` function? Flake8 is starting to find this function to complex now.

Comment (TotalCaesar659) [reply minigalaxy/ui/window.py]: Fixed.

**Comment by TotalCaesar659** [comment]
Hm, I don't see this exception so I don't know what to do with it, please retest.

**Comment by jgerrish** [comment]
Content-length isn't a required header:

[14.13 Content-Length](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html)

So, maybe just a test for None before casting?

I don't know why it's failing sometimes.  Interrupted connections, GOG proxies running different versions or with different views of the cache, I don't know.  

I saw the behavior a few times myself.

**Pull request merged**